sdk/search/search-documents/test/utils/setupIndex.ts

```diff
@@ -51,7 +51,7 @@ export async function createIndex(client: SearchIndexClient, name: string): Prom
         facetable: true
       },
       {
-        type: "Collection(Edm.String)",
+        type: "Edm.String",
```

Comment:
why are these not collections anymore?

Reply:
This is the result of the latest PR: https://github.com/Azure/azure-rest-api-specs/pull/9301/files#diff-a01e10da9dba0149960db35a9d106976L2632. As a result the collection is gone from the generated code. As a result, I removed it from the custom code too

Reply:
Collection types were never in the swagger, the libraries have always had to add them in manually. We shouldn't remove these from our code.